spec/controllers/admin/domain_blocks_controller_spec.rb

```diff
@@ -8,17 +8,23 @@
   end
 
   describe 'GET #index' do
-    it 'returns http success' do
-      get :index
+    it 'renders domain blocks' do
+      2.times { Fabricate(:domain_block) }
+
+      get :index, params: { page: 2 }
 
+      assigned = assigns(:domain_blocks).per(1)
```

Comment:
Putting this `per` in the spec seems weird ... b/c the next line about `count` eq 1 is now just checking the result of something done right here, not done in the controller.

You could set `DomainBlock.paginates_per = 1` in an `around` here (and then set back after spec), which seems better?